inspirehep/utils/citations.py

```diff
@@ -42,8 +44,7 @@ def citations(self):
 
         out = []
         row = []
-        recid = self.record['control_number']
-        es_query = Query('refersto:' + str(recid)).search()
+        es_query = Query('refersto:' + str(self.recid)).search()
```

Comment:
@jmartinm, no it won't affect the way we do the caching. It is quite likely that less single-entry views will be cached after changes :smile:  Are we also going to display max 10 references of the record? Finally, would you prefer to do `len()` on the client side or rather have it provided within the AJAX response?